lib/danger/ci_source/buddybuild.rb

```diff
@@ -1,35 +1,54 @@
 module Danger
   # ### CI Setup
   #
-  # Buddybuild has an integration for Danger already built-in.
-  # What you need to do is to upload your `Gemfile` and `Dangerfile` to
-  # the server and you should be all set-up. However, if you want to use
-  # different bot for Danger, you can do it with token setup described below.
-  #
```

Comment:
I've updated the docs in the to include a link to our documentation. I'll open a new pull request once the documentation is out.